006-vortexa-tech-challenge/src/map.tsx

```diff
@@ -0,0 +1,87 @@
+import "maplibre-gl/dist/maplibre-gl.css";
+
+import { Feature, MultiPolygon } from "geojson";
+import { RampData, RampProperties } from "./types";
+import React, { Dispatch, SetStateAction, useState } from "react";
+import ReactMap, { Marker, Popup, ViewStateChangeEvent } from "react-map-gl";
+
+import maplibregl from "maplibre-gl";
```

Comment:
Similarly here, sort the imports.

```suggestion
import React, { Dispatch, SetStateAction, useState } from "react";
import maplibregl from "maplibre-gl";
import { Feature, MultiPolygon } from "geojson";
import ReactMap, { Marker, Popup, ViewStateChangeEvent } from "react-map-gl";
import "maplibre-gl/dist/maplibre-gl.css";

import { RampData, RampProperties } from "./types";
```